docs/ipsec-tunnel.md

```diff
@@ -0,0 +1,52 @@
+# IPsec Encryption of Tunnel Traffic with Antrea
+
+Antrea supports encrypting tunnel traffic across Nodes with IPsec ESP. At this
+moment, IPsec encyption works only for GRE tunnel (but not VXLAN, Geneve, and
+STT tunnel types).
+
+## Prerequisites
+
+IPsec requires a set of Linux kernel modules. Check the required kernel modules
+listed in the [strongSwan documentation](https://wiki.strongswan.org/projects/strongswan/wiki/KernelModules).
+Make sure the required kernel modules are loaded on the Kubernetes Nodes before
+deploying Antrea with IPsec encyption enabled.
+
+## Installation
+
+You can simply apply the [Antrea IPsec deployment yaml](/build/yamls/antrea-ipsec.yml)
+to deploy Antrea with IPsec encyption enabled. To deploy a released version of
+Antrea, pick a version from the [list of releases](https://github.com/vmware-tanzu/antrea/releases).
+For any given release `<TAG>` (e.g. `v0.3.0`), you can get the Antrea IPsec
+deployment yaml at:
+```
+https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-ipsec.yml
+```
+
```

Comment:
I suggest making it clear that IPsec support wasn't available before the 0.3.0 release by adding here:

> Note that IPsec support was added in release 0.3.0, which means the `<TAG>` you select cannot correspond to an older release.

Reply:
Sure.